examples/rds/dbparametergroup.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rds.aws.crossplane.io/v1alpha1
+kind: DBParameterGroup
+metadata:
+  name: example-dbparametergroup
+spec:
+  forProvider:
+    region: eu-central-1
+    dbParameterGroupName: example-dbparametergroup
```

Comment:
```suggestion
```

Reply:
Why do we need to remove the dbParamterGroupName and how to exactly do this? if i run make generate, it is not removed

Reply:
Also,  it seems that i am not authorized to squash & merge due to the branch being protected. 

make reviewable test was green as well as create, update and delete scenarios.

Reply:
You can squash your commits locally and then `git push --force`  because you’re pushing to your own fork